scripts/openzfs-merge.sh

```diff
@@ -106,6 +109,9 @@ generate_desc() {
 #add description to commit
 add_desc_to_commit() {
 	git commit --amend -F "$TMP_FILE"
+	if [ $PUSH_NEW_BRANCH -ne 0 ]; then
```

Comment:
I would suggest moving this out into the merge function. You can do something like
```
if [ $PUSH_NEW_BRANCH -ne 0 ] || [ $ERR -eq 0 ]; then
# push branch here
fi
```

Then you can remove the git push from the existing ERR check.